.gitignore

```diff
@@ -12,3 +12,4 @@ npm-cache/
 .ocp-port-forwarding.js
 *.log
 .rpt2_cache
+package-lock.json
```

Comment:
Why add it to the .gitignore? Don't we need it for the CI step, to make sure we install the exact version?

Reply:
We don't use npm for the monorepo, we use yarn, so does the CI. We had issues with mom and yarn is super fast in comparison